gateway/handler/cluster.go

```diff
@@ -33,19 +33,22 @@ func NewCluster(p Params) *Cluster {
 	}
 }
 
-func (h *Cluster) CreateCluster(c *gin.Context) {
-	var createClusterRequest request.CreateClusterRequest
-	if err := c.ShouldBindJSON(&createClusterRequest); err != nil {
-		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": ErrorBadRequest})
-		return
+func (h *Cluster) getCustomer(c *gin.Context) *models.Customer {
+	//debug
+	if claims, exists := c.Get("claims"); exists {
+		claims := claims.(*models.AuthJwtClaims)
+		fmt.Println("-----------------------")
```

Comment:
We should remove the print statements before the PR is merged or at least changing to log statements that they are in line with the other log outputs.